packages/pharos/src/components/dropdown-menu/pharos-dropdown-menu-item.scss

```
@@ -68,6 +68,7 @@
   margin-right: var(--pharos-spacing-one-half-x);
 }
 
+:host([disabled]),
 :host([disabled]) .dropdown-menu-item__link,
 :host([disabled]) .dropdown-menu-item__button {
```

I believe if an element has `pointer-events: none`, its children will also have `pointer-events: none` unless otherwise specified. May be able to simplify this to simply:

```suggestion
:host([disabled]) {
```